### DISCUSSION TITLE
is removing capacitor means we are no longer will have mobile app ?

### DISCUSSION THREAD

**Discussion by yahyaoudra** [The-Commit-Company/raven#595] [General]
I saw on ticket #591  that you guys removed capacitor is that means mobile app will no longer will be available ? @nikkothari22

**Comment by nikkothari22** [comment]
No no. We just removed the capacitor CLI package which wasn't required. Mobile app is definitely there - we don't need capacitor for it - we ourselves won't be able to use Raven otherwise.

**Reply by yahyaoudra** [reply]
Thank you @nikkothari22 for explaining and thank you for this discussion space better than opening issues 😅
